.circleci/config.yml

```diff
@@ -331,6 +340,25 @@ jobs:
           name: docs
           command: sh scripts/kdoc-validate.sh
 
+  verify-api:
+    docker:
+      - image: mbgl/android-ndk-r21e:latest
+
+    steps:
+      - checkout
+      - restore-gradle-cache
+      - init-aws
+      - run:
+          no_output_timeout: 30m
```

Comment:
that is a long time :smile: :see_no_evil:

Reply:
In the CI, it actually cost 12m 26s 😄